src/@types/event.ts

```diff
@@ -144,6 +144,15 @@ export const UNSTABLE_MSC3089_LEAF = new UnstableValue("m.leaf", "org.matrix.msc
  */
 export const UNSTABLE_MSC3089_BRANCH = new UnstableValue("m.branch", "org.matrix.msc3089.branch");
 
+/**
+ * Functional members type for declaring a purpose of room members (e.g. helpful bots).
+ * Note that this reference is UNSTABLE and subject to breaking changes, including its
+ * eventual removal.
+ */
+export const UNSTABLE_ELEMENT_FUNCTIONAL_USERS = new UnstableValue(
+    "io.element.functional_members",
+    "io.element.functional_members");
```

Comment:
I'd sooner hop into `UnstableValue` and mark the second field as optional, fixing any docs along the way.

Reply:
I'd make it a type of its own, as UnstableValue has a different meaning, is exported and also requires `unstable` to be defined.

Or do you mean to set `.unstable` to the same value as `.name` if no second parameter is provided to the constructor?